modules/codemod/lib/v5/spec/compoundCard.spec.ts

```diff
@@ -0,0 +1,169 @@
+import {expectTransformFactory} from './expectTransformFactory';
+import transform from '../compoundCard';
+import {stripIndent} from 'common-tags';
+
+const expectTransform = expectTransformFactory(transform);
+
+describe('card', () => {
+  it('should restructure Card usages with a literal heading', () => {
+    const input = stripIndent`
+      import {Card} from '@workday/canvas-kit-react/card'
+
+      <Card heading="Card Heading" headingId={id}>Card Contents</Card>
+    `;
+
+    const expected = stripIndent`
+      import {Card} from '@workday/canvas-kit-react/card'
+
+      <Card><Card.Header id={id}>Card Heading</Card.Header><Card.Body>Card Contents</Card.Body></Card>
+    `;
+
+    expectTransform(input, expected);
+  });
+
+  it('should restructure Card usages with no props', () => {
+    const input = stripIndent`
+      import {Card} from '@workday/canvas-kit-react/card';
+
+      <Card>
+        No Padding
```

Comment:
Nit: this text doesn't really make sense in this context. It's used in all the tests.

Reply:
I prefer explicit testing vs implicit testing. If this test fails, it is explicitly for the lack of props where the other tests have additional failure points

Reply:
I think I see what you're saying, @NicholasBoll, but doesn't the default Card have padding though?
https://github.com/Workday/canvas-kit/pull/1028/files#diff-c9e6ab3d13769a201aedc2ac98abcf34e296920d943bc799a372f392f8323174R64

Reply:
Maybe the text should read "No props" instead?

Reply:
Yes, I was just referring to the text inside the card, not the test itself